tests/testthat/_snaps/geodist.md

```diff
@@ -106,7 +107,8 @@
     Condition
       Error in `step_geodist()`:
       Caused by error in `prep()`:
-      ! `lon` should resolve to a single column name.
+      x The `lon` selector should select at most a single variable.
+      i The following 2 were selected:`y` and `y1`.
```

Comment:
```suggestion
      i The following 2 were selected: `y` and `y1`.
```

Reply:
done in https://github.com/tidymodels/recipes/pull/1255/commits/483af8a84507657d909bec152f8bb87e15fb9350